packages/directory/app/styles/navi-directory/components/dir-table.less

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+.dir-table {
+  overflow-x: auto;
+  padding: 0 20px;
+
+  thead {
```

Comment:
Is it possible to have classNames for these?

Reply:
Not from what I found in ember-light-table. These styles are only applied to children of `dir-table` so it should be okay.

Reply:
Update: found a way to give the cells and rows custom class names